.github/workflows/elementor-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Elementor Tests / Init
+
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@master
+      - name: Block non-permitted users
+        uses: StyleShit/action-whitelist-users@master
+        with:
+          whitelist-file: '.github/workflows/whitelist.txt'
+      - name: Elementor Tests / Init
+        env:
+          TOKEN: ${{ secrets.DEVOPS_TOKEN }} # GitHub access token to access the remote repo.
```

Comment:
```suggestion
          TOKEN: ${{ secrets.MAINTAIN_TOKEN }} # GitHub access token to access the remote repo.
```

Reply:
Done.

Reply:
Looks like the token doesn't exist.
https://github.com/elementor/elementor/runs/2466428236?check_suite_focus=true

(Open to see the parameters, they are empty)